src/components/FormikForm/FormikForm.css

```diff
@@ -80,7 +80,7 @@
       }
 
       .bp3-tag-input-values {
-        margin: 1px 7px 2px 1px !important;
+        margin: 2px 7px 0 1px !important;
```

Comment:
Possible to center align without margin? 

We will have to change these values if in any case height of the input changes

Reply:
Not possible right now.

Also, blueprintjs internally has a min-height of 30px for tag input. If ever needed, we'll have to overwrite blueprint's styles too.